### PR TITLE
(GH-3) Update Cake.Bridge to target Cake v1.0.0

### LINKED ISSUE #3: Update Cake.Bridge to target Cake v1.0.0
https://cakebuild.net/blog/2021/02/cake-v1.0.0-released

:relaxed:

### PR DESCRIPTION
**Pull request by augustoproiete**
Closes #3

- Updates Cake.Bridge to target Cake v1.0.0

### CODE REVIEW

File: src/Cake.Bridge/Cake.Bridge.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>net5.0;netstandard2.0;net461</TargetFrameworks>
```
Comment (devlead) [inline src/Cake.Bridge/Cake.Bridge.csproj]: `Cake.Core` / `Common` still targets `net46` so see no need to potentially break things by changing this.

**Review by devlead — CHANGES_REQUESTED** [review]
https://github.com/devlead/Cake.Bridge/blob/e538bc5f8f0941d2fc650226b700b0a7199690b5/src/cake.ps1#L21

The PowerShell helper will also need to reflect the new Cake version.

File: src/Cake.Bridge/Cake.Bridge.csproj (hunk 1 above)
Comment (augustoproiete) [reply src/Cake.Bridge/Cake.Bridge.csproj]: Reverted && push --force

**Comment by augustoproiete** [comment]
Ready for round 2

**Review by devlead — APPROVED** [review]
LGTM 👍 

Reverted the change to root `build.ps1` as that needs to be updated when using  a new version of Bridge.

**Comment by devlead** [comment]
@augustoproiete  your changes have been merged, thanks for your contribution 👍

**Pull request merged**